src/components/Header.module.scss

```diff
@@ -0,0 +1,17 @@
+.main {
+  color: var(--background-light);
+  background-color: var(--main);
+  padding-top: var(--x);
+  padding-bottom: var(--x);
+  margin-bottom: var(--x);
```

Comment:
Some elements in the header still use the '--x' variable instead of '--bootstrap-padding'. We might want to unify all paddings in the future.

Reply:
Yes. Ideally we would rarely need that variable and instead properly use the `Row` and `Col` components from `reactstrap` to take advantage of the bootstrap grid. I mainly introduced `--bootstrap-padding` for overriding the bootstrap grid on the home page where we don't want to have that padding for the carousel.